BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -22,12 +41,18 @@ def __create_event(self, timestamp):
 
     def update_state(self, timestamp: str, status_code: str):
         """
-        Updates the state of the signal group.
 
+        Updates the state of the signal group.
         Returns a tuple representing a traffic light cycle
         event (RED to GREEN to RED) if one was completed
         as a result of the update.
         Otherwise returns None.
+        # Parameter:
+        # argument1: timestamp(String)
+        # argument2: status_code(String)
+        # Returns:
+        # Event (TrafficLightCycle)
```

Comment:
Returns either `None`, if no events were completed as a result of updating the state, or `Tuple[str, str, str, str, str]` (device id, signal group, string representing the red light start time, string representing the green light start time, string representing the green light end time).